### ISSUE TITLE
Stopped working in Thunderbird 128

### ISSUE THREAD

**Issue by nowrap** [mganss/AttachFromClipboard#15]
AttachFromClipboard options are not showing in Thunderbrid 128.0.

**Comment by mganss** [comment]
Fixed in 1.128.3 (currently under review at Mozilla).

**Comment by R00118189** [comment]
Hi, I am on on TB 128.1.0esr (64-bit)
Add in version is 1.128.4 Last Updated 19 August 2024 installed directly from TB or from file
All other add-ins disabled - no difference

No action when "paste from Clipboard" triggered either from the dropdown or menu bar

While I have limited knowledge about debugging TB I opened the debug add-in and got
Paused on exception : TypeError: Cc['@mozilla.org/embedcomp/prompt-service;1'] is undefined in line 106 of overlay.js

```
var promptService = Cc["@mozilla.org/embedcomp/prompt-service;1"].getService(Ci.nsIPromptService);
```

![image](https://github.com/user-attachments/assets/895fc9dc-33eb-40ae-9f88-85533577b243)
![image](https://github.com/user-attachments/assets/7a8ca87a-ddd2-4f89-a015-de4b4f2b795f)

forgot to add stack

![image](https://github.com/user-attachments/assets/45e57fef-5dea-480e-8cf3-9433d4d65f63)
and console
![image](https://github.com/user-attachments/assets/ac0f6f1d-ffd7-49cb-ae00-c2544e950cb3)
